src/App.js

```diff
@@ -1,63 +1,88 @@
 import './App.css';
 import VehicleList from './VehicleList';
 import TrafficLight from './TrafficLight';
-
+import { useState } from 'react';
 function App() {
   // track the following state with a few useState hooks:
   // lightColor should be a string that starts out as 'red'
+  const [lightColor, setLightColor] = useState('red');
   // lizardSize should be a number that starts out as 10
+  const [lizardSize, setLizardSize] = useState(10);
   // alienSize should be a number that starts out as 10
+  const [alienSize, setAlienSize] = useState(10);
+  const SIZE_MODIFIER = 1;
   // traffic is complicated. It should be an array of strings that starts out as ['car', 'truck']
+  const [vehicle, setVehicle] = useState(['car', 'truck']);
+
+  function handleCar(){
+    vehicle.push('car');
+    setVehicle([...vehicle]);
```

Comment:
for all of these vehicle functions we can avoid doing the push on line 18 by instead having line 19 say `setVehicle([...vehicle, 'car']);` and obviously making `handleBus()` be inserting 'bus' instead of car, etc.